code/modules/jobs/job_types/quartermaster.dm

```diff
@@ -1,20 +1,26 @@
 /datum/job/qm
 	title = "Quartermaster"
 	flag = QUARTERMASTER
-	department_head = list("Head of Personnel")
+	auto_deadmin_role_flags = DEADMIN_POSITION_HEAD
+	department_head = list("Captain")
```

Comment:
below, it says that "absolutely no one" is the QM's supervisor, but this seems to contradict that

Reply:
```suggestion
	department_head = null // NO GODS, NO MASTERS
```

Reply:
i think i need to put a job there